mysql/tests/test_metadata.py

```diff
@@ -33,8 +32,13 @@ def sort_names_split_by_coma(names):
 def normalize_values(actual_payload):
     actual_payload["default_character_set_name"] = "normalized_value"
     actual_payload["default_collation_name"] = "normalized_value"
+    actual_payload["tables"].sort(key=lambda x: x["name"])
     for table in actual_payload['tables']:
         table['create_time'] = "normalized_value"
+        if 'columns' in table:
+            table['columns'].sort(key=lambda x: x['name'])
+        if 'indexes' in table:
+            table['indexes'].sort(key=lambda x: x['name'])
```

Comment:
Adding the sorting here due to flaky tests. I think the only time ordering will matter is in the case of columns, but they have an ordinal_position field, so deterministic ordering in the payload should be unnecessary.